docs/reference/search/field-caps.asciidoc

```diff
@@ -137,11 +137,11 @@ field types are all described as the `keyword` type family.
 
 `time_series_dimension`::
   preview:[]
-  Whether this field is used as a time series dimension.
+  Whether this field is used as a time series dimension on all indices.
```

Comment:
@kilfoyle Maybe you have a better take on the doc changes here.

Reply:
Looks good now! 👍